home/modules/contribute/pages/extensions-embargoed.adoc

```diff
@@ -0,0 +1,5 @@
+:page-embargo: EMBARGOED
```

Comment:
Can this page attribute line be wrapped within `ifdef` / `ifndef` directives?

(I'm sure it can, just checking)

Reply:
Yes! But you're right, that'd be a much better and more reassuring test... 👍 
I'll add another example page shortly...